### PR TITLE
chore: constant rollkit ignite app version

### PR DESCRIPTION
**Pull request by yarikbratashchuk**
## Summary by CodeRabbit

- **New Features**
  - Updated tutorials and guides to use dynamic versioning for installation scripts.
  
- **Documentation**
  - Improved installation commands across tutorials to dynamically reference version constants.

### CODE REVIEW

File: .vitepress/constants/constants.js [hunk 1]
```diff
@@ -4,9 +4,10 @@ const constants = Object.freeze({
   nodeVersion: "21.7.2",
   yarnVersion: "1.22.19",
 
-  rollkitLatestTag: "v0.13.3",
+  rollkitLatestTag: "v0.14.3",
```
Comment (Manav-Aggarwal) [inline .vitepress/constants/constants.js]: should be v0.13.4: https://github.com/rollkit/rollkit/releases/tag/v0.13.4

File: tutorials/cosmwasm.md [hunk 2]
```diff
@@ -29,8 +30,8 @@ this tutorial.
 
 For this tutorial, we will need `go` and `jq` installed on your machine. You can install them by running our script:
 
-```bash
-curl -sSL https://rollkit.dev/install-go.sh | bash -s go1.22.3
+```bash-vue
+curl -sSL https://rollkit.dev/install-go.sh | bash -s {{constants.golangVersion}}
```
Comment (Manav-Aggarwal) [inline tutorials/cosmwasm.md:34]: This seems to be getting rendered incorrectly at least in https://rollkit.dev/guides/connect-local-da, please make sure that's not the case.

**Review by Manav-Aggarwal — CHANGES_REQUESTED** [review]
Left comments

File: .vitepress/constants/constants.js (hunk 1 above)
Comment (yarikbratashchuk) [reply .vitepress/constants/constants.js]: yeah, realized that

File: tutorials/cosmwasm.md (hunk 2 above)
Comment (yarikbratashchuk) [reply tutorials/cosmwasm.md:34]: agh,yes

**Review by jcstein — APPROVED** [review]

**Review by Manav-Aggarwal — APPROVED** [review]

**Pull request merged**